### PR TITLE
Updated Changi TestNet Guides

### PR DESCRIPTION
**Pull request by virgchiniwala**
Updated both Changi guides and removed the old Floppynet guides

### CODE REVIEW

File: src/guides/guide_changi.md [hunk 1]
```diff
@@ -32,7 +28,7 @@ defid -changi
 >
 > `RUST_LOG=debug defid -changi --daemon`
 
-Once the node is running it should start syncing. The process of syncing can take up to a couple of hours depending on your machine so just be patient.
+Once the node is running it should start syncing. The process of syncing can a couple of hours depending on your machine so just be patient.
```
Comment (prasannavl) [inline src/guides/guide_changi.md:31]:
```suggestion
Once the node is running it should start syncing. The process of syncing can take a couple of hours depending on your machine.
```

File: src/guides/guide_changi.md [hunk 2]
```diff
@@ -44,11 +40,9 @@ defi-cli getblockchaininfo | head
 
 ### Connect Metamask.
 
-To connect to MetaMask the process is the same as the one described in [Connect Metamask to Floppynet](./guide_floppynet_short.md) and in [point 4](./guide_floppynet_short.md#4-fill-network-form-with-connection-data) use the Changi Testnet connection data below. For now only the Chain ID and the optional block explorer in the Changi Testnet are different:
+To connect to MetaMask the process is the same as the one described in [Connect Metamask to Changi TestNet](./guide_changi_short.md) you can refer to that guide for more information on it. 
+> Do note that the RPC url will need to be pointed to your local node and not http://changi.dfi.team
```
Comment (prasannavl) [inline src/guides/guide_changi.md:44]:
```suggestion
> Do note that the RPC url will need to be pointed to your local node (Default: `http://127.0.0.1:20551`) and not http://changi.dfi.team
```

File: src/guides/guide_changi_short.md [hunk 3]
```diff
@@ -24,10 +24,11 @@ Connection data:
 - Network name: **MetaChain**
 > ⚠️ **Warning: Due to potential network overloads, it is advised to use a local node over the public node to connect with Metamask. Follow the guide: [Run a Local Node in Changi](./guide_changi.md).**
 
-- RPC URL: https://changi.dfi.team
-- Chain ID: **1133** (Specific to Changi TestNet)
-- Currency symbol: **DFI**
-- Block explorer URL: https://meta.defiscan.live
+| Network | RPC URL | Chain ID | Currency symbol | Block Explorer (Optional) |
+|---------|---------|----------|----------------|----------------|
+|Changi Testnet| http://changi.dfi.team| 1133 | DFI | https://meta.defiscan.live |
```
Comment (prasannavl) [inline src/guides/guide_changi_short.md:29]:
```suggestion
|Changi Testnet / Managed Node| https://changi.dfi.team | 1133 | DFI | https://meta.defiscan.live |
|Changi Testnet / Local Node | https://127.0.0.1:20551 | 1133 | DFI | https://meta.defiscan.live |
```

Please note that the managed node are public nodes run by the core-dev team for convenience and that they may be susceptible to rate limits.

File: src/guides/guide_changi_short.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-# Connect Metamask to Changi TestNet
+![252304387-ec0f54dc-908b-4a01-8a7c-e0e778ddef09](https://github.com/DeFiCh/handbook/assets/113342097/d5273de6-eba3-43ae-b8e7-1f39f0373830)# Connect Metamask to Changi TestNet
```
Comment (shohamc1) [inline src/guides/guide_changi_short.md:1]: Better to link the image locally instead

Comment (virgchiniwala) [reply src/guides/guide_changi_short.md]: done @shohamc1

File: src/guides/guide_changi_short.md [hunk 5]
```diff
@@ -4,17 +4,17 @@
...
 #### 1. Open network dropdown.
 <p align="center">
-  <img width="356" alt="step1" src="https://github.com/DeFiCh/handbook/assets/113342097/ab62ae59-a140-45ff-a1e8-17db0c0a8683">
+  <img width="356" alt="step1" src="https://github.com/DeFiCh/handbook/blob/virgc/testnet_doc/src/guides/screenshots/step1_guide_changi_short.png">
 </p>
 
 #### 2. Add network
 <p align="center">
-<img width="354" alt="step2" src="https://github.com/DeFiCh/handbook/assets/113342097/64a24837-114c-406f-b624-63f4974677fa">
+<img width="354" alt="step2" src="https://github.com/DeFiCh/handbook/blob/virgc/testnet_doc/src/guides/screenshots/step2_guide_changi_short.png">
```
Comment (shohamc1) [inline src/guides/guide_changi_short.md:12]: Need to link to the local file. You can also remove the centre align when using this way to link images.

```suggestion
![](./screenshots/step2_guide_changi_short.png)
```

**Review by shohamc1 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**